infra/modules/azure_mysql_server/main.tf

```diff
@@ -0,0 +1,68 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = ">= 3.116.0"
```

Comment:
Is this very high version mandatory also here?

Reply:
actually no, now I installed `3.111` which doesn't seem to cause any problems